lib/log.rb

```diff
@@ -80,6 +80,33 @@ def initialize(level=nil)
       else
         @level = INFO
       end
+      @ignore_log = false
```

Comment:
Lines 83-85, missed indents.

Reply:
others, I'm OK.

Reply:
Updated and also added addition comments.